test/parse.js

```diff
@@ -6,8 +6,11 @@ var cookie = require('..');
 suite('parse');
 
 test('argument validation', function() {
+    var nonStrings = [1010, true, {}, (function() {return;}), new Date(), []];
```

Comment:
Please just do one `assert.throws` line for each instead of iterating an array; an error won't tell you which value was allowed though, only point to line 12, which is not helpful.